### ISSUE TITLE
Quest 27874 "Aviana's Legacy" not showing as available although it is

### ISSUE THREAD

**Issue by TorpedoCXC** [Questie/Questie#6681]
### Description

https://www.wowhead.com/cata/quest=27874/avianas-legacy

![WoWScrnShot_111824_150535](https://github.com/user-attachments/assets/054ddcaa-9f6d-434f-b00f-87629c8ca763)
![WoWScrnShot_111824_150615](https://github.com/user-attachments/assets/19bbfd19-2c7d-4c58-8f00-59ae4c3bcea0)

This is a breadcrumb quest for https://www.wowhead.com/cata/quest=25663/an-offering-for-aviana

### Questie version

v10.9.2

### Game flavor

Cata

**Comment by Cabro** [comment]
OK, that quest has so many quest starters, not even wowhead has them all :D

**Comment by Cabro** [comment]
What's the ID for that Ysera? 40289?

**Comment by TorpedoCXC** [comment]
Yes, https://www.wowhead.com/cata/npc=40289/ysera seems correct.

I don’t remember which quest I did before. I was already very near the achievement for completing the zone though.

**Comment by TorpedoCXC** [comment]
According to another plugin I am using (which is usually right about these things), one has to complete https://www.wowhead.com/cata/quest=25612/return-from-the-firelands or https://www.wowhead.com/cata/quest=25611/return-from-the-firelands before it becomes available.

**Comment by Cabro** [comment]
![image](https://github.com/user-attachments/assets/f5a85b20-7212-46be-8f41-1c8dde02a738)
That's exactly what i also noticed ingame. How come it didn't have tooltip for you?

**Comment by TorpedoCXC** [comment]
I strongly suspect the exclusiveTo is wrong. I remember doing the quest before doing 27874.

**Comment by TorpedoCXC** [comment]
I just had a look into the journal. I completed 25985 before accepting 27874.

![grafik](https://github.com/user-attachments/assets/1a39acb8-95e8-489f-a15e-05d378b4625d)
